IPs/RSKIP208.md

```diff
@@ -0,0 +1,59 @@
+# checkEnvironment Precompile method
+
+
+|RSKIP          | 208 |
+| :------------ |:-------------|
+|**Title**      |checkEnvironment Precompile method|
+|**Created**    |JAN-2021 |
+|**Author**     |SDL |
+|**Purpose**    |Usa |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+
+# **Abstract**
+
+This RSKIP proposes to add a method `checkEnvironment()` to a new `Environment`  pre-compile contract. The new method receives as argument a list of fields that represents the minimum environment requirements to execute a method and it checks that the requirements are satisfied. This RSKIP provides the same functionality of [RKIP203](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP203.md), but in a way which allows the future addition of additional environment variables, such as rent gas limit, for storage rent accounting.
+
+## Motivation
+
+The motivation is the one described in [RSKIP203](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP203.md) plus we add the requirement that contracts that forward contracts do not become vulnerable if another counter of any kind is added to the environment.
+
+This RSKIP proposes to create a new `Environment` pre-compile contract to with a method to check if the minimal environment requirements given as the `env` argument are satisfied. The argument contains a list of fields in RLP format. This RSKIP defines that the first argument of the list is the stack depth. The gas limit is left outside `env` because the caller must provide the value externally to `CALL`. The remaining arguments are undefined and are ignored. Users and relayers will negotiate a content for `env`. Smart contract do not need to interpret the content of `env`, and can simply pass it to the `checkEnvironment()` method.
```

Comment:
Remove **to** from "pre-compile contract to with a "

Reply:
ok